### ISSUE TITLE
Menril and Ultimine

### ISSUE THREAD

**Issue by HalJordan** [ChosenArchitect/Project-Architect-2#40]
It still eludes me as to how this isn't in ultimine by default SINCE THEY HAVE A CLOSED ISSUE ON IT that should be fixing it, unless everyone is using a years old version still???

ftbultimine-server.snbt needs the menril_logs tag added so that you can chop the WHOLE tree at once:

```
	# These tags will be considered the same block when checking for blocks to Ultimine in shapeless mining mode
	# Default: ["minecraft:base_stone_overworld", "c:*_ores", "forge:ores/*"]
	merge_tags: [
		"c:*_ores"
		"forge:ores/*"
		"integrateddynamics:menril_logs"
	]
```

Note, I remove the stone one so that I can ultimine ONLY granite/diorite/andesite/etc when I'm looking for those specifically.

**Comment by shiftthedev** [comment]
unable to replicate this issue in modpack version 2.6
using an axe with ultimine works in menril logs

**Comment by HalJordan** [comment]
Did it catch the everyday ones too?

I literally JUST installed the latest pack available and had to fix this
manually. Just as I've had to do for EVERY pack with both in it.

**Comment by shiftthedev** [comment]
ok I think I misunderstand the issue.
you want to be able to ultimite Menril Logs and Enriched Menril Logs as they were the same?
I think that we can add that QOL.

**Comment by HalJordan** [comment]
It's just the one line in that sbt file if I read ultimine's GitHub
correctly it 'should' already be there, but out of a dozen or so packs I've
never seen any include that line in the config.

**Comment by IRicTheCoder** [comment]
This quality of life has been added and should be available on 4.0+, closing this as solved.
